src/System.Numerics.Vectors/tests/GenericVectorTests.tt

```diff
@@ -334,7 +334,8 @@ namespace System.Numerics.Tests
             int expected = 0;
             for (int g = 0; g < Vector<T>.Count; g++)
             {
-                expected = (((expected << 5) + expected) ^ v1[g].GetHashCode());
+			    uint shift5 = ((uint)expected << 5) | ((uint)expected >> 27);
```

Comment:
nit: indentation is a bit off here. These t4 templates are hard to format, I know :smile:. Try to make sure you replace tabs with spaces in this file. Visual Studio doesn't use the same text formatting options in .tt files, if that is what you're editing with, so you'll have to manually replace the tabs.